nisystemlink/clients/spec/_spec_client.py

```diff
@@ -0,0 +1,97 @@
+"""Implementation of SpecClient"""
+
+from typing import Optional
+
+from nisystemlink.clients import core
+from nisystemlink.clients.core._uplink._base_client import BaseClient
+from nisystemlink.clients.core._uplink._methods import get, post
+
+from . import models
+
+
+class SpecClient(BaseClient):
+    def __init__(self, configuration: Optional[core.HttpConfiguration]):
+        if configuration is None:
+            configuration = core.JupyterHttpConfiguration()
+        super().__init__(configuration, base_path="/nispec/v1/")
+
+    @get("")
+    def api_info(self) -> models.V1Operations:
+        """Get information about available API operations.
+
+        Returns:
+            Information about available API operations.
+
+        Raises:
+            ApiException: if unable to communicate with the DataFrame Service.
+        """
+        ...
+
+    @post("specs")
+    def create_specs(
+        self, specs: models.CreateSpecificationsRequest
+    ) -> models.CreateSpecificationsPartialSuccess:
+        """Creates one or more specifications.
+
+        Args:
+            specs: A list of specifications to create.
+
+        Returns:
+            A list of specs that were successfully created and ones that failed to be created.
+
+        Raises:
+            ApiException: if unable to communicate with the `/nispec` service or if there are
+            invalid arguments.
+        """
+        ...
+
+    @post("delete-specs")
+    def delete_specs(
+        self, spec_ids: models.DeleteSpecificationsRequest
+    ) -> Optional[models.DeleteSpecificationsPartialSuccess]:
```

Comment:
For routes with simple request bodies like this, we don't need to create a model. You can follow the pattern here:
https://github.com/ni/nisystemlink-clients-python/blob/cd342320b76ee9179628a87f357591500c828717/nisystemlink/clients/dataframe/_data_frame_client.py#L165-L168

Reply:
Nice. That is much simpler. I've made that change for delete specs.